playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -0,0 +1,97 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- name: Copy skuba openstack terraform files to socok8s workspace
+  copy:
+    src: "{{ playbook_dir }}/../submodules/skuba/ci/infra/openstack/"
```

Comment:
Did you intend to add this as a submodule?

Reply:
Yes (see the commit messages). Currently skuba is a private repo so I can not add it as a submodule. But the skuba maintainers plan to make it public (in the next week(s)) and then we can switch to a submodule.

Reply:
I would assume ideally we'd like to install it from package...

Reply:
If we want to use the terraform form, I guess we need to use a submodule or something similar. Not sure if terraform forms will be available via packages.
For the `skuba` executable, I agree (but for now I would just build it from git)